src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEParameterSymbol.cs

```diff
@@ -104,7 +104,7 @@ public bool SetWellKnownAttribute(WellKnownAttributeFlags flag, bool value)
                 if (value)
                 {
                     // the actual value:
-                    bitsToSet |= ((int)flag << WellKnownAttributeDataOffset);
+                    bitsToSet |= (int)flag << WellKnownAttributeDataOffset;
```

Comment:
Parens are welcome when using unusual operator.

Reply:
Yup.  This is the same case as the first one.  When assignment and anything else are mixed.  Just as an FYI @gafter , these are the defaults i'm recommending we ship with:

![image](https://user-images.githubusercontent.com/4564579/37569259-ff8f353a-2a9c-11e8-8acd-3978b663d47b.png)

Meaning, by default, we'll actually ignore almost all parens in code, allowing people to just use them as they want.  However, for the 'other' category, we will slightly suggest they remove the parens *just* by fading them out.  You won't get anything else in the IDE by default aggressively pointing this out.  

So what is the 'other' category.  Right now, it's any cases where parens are used outside of binary/ternary expressions and where we can tell they're superflous.  So, for example things like ```(x.y).length``` or ```case (x.y):```  or ```if ((x.y > 10))``` etc. etc.  

It does seem like assignment though is a special case.  It's not a binary, but it is similar enough to warrant special attention.